skymp5-server/cpp/server_guest_lib/FormDesc.cpp

```diff
@@ -1,24 +1,38 @@
 #include "FormDesc.h"
-#include <sstream>
+#include <cstdio>
 
 std::string FormDesc::ToString(char delimiter) const
 {
-  std::stringstream ss;
-  ss << std::hex << shortFormId;
-  if (!file.empty())
-    ss << delimiter << file;
-  return ss.str();
+  auto fmt = "%0x%c%s";
+  size_t size = !file.empty()
+    ? std::snprintf(nullptr, 0, fmt, shortFormId, delimiter, file.c_str()) + 1
```

Comment:
As far as I know +1 isn't needed
`std::string` already adds +1 byte for `\0` char